drp.tex

```diff
@@ -138,88 +133,88 @@ \subsubsection{Coaddition}
+The resulting Sersic model fluxes are provided as an alternative to CModel and are intended to represent total galaxy fluxes.
 Like CModel, the Sersic model is a Gaussian mixture approximation to a true Sersic profile, convolved with a Gaussian mixture approximation to the \gls{PSF}.
 CModel measurements use a double ``shapelet'' \citep{2003ARA&A..41..645R} PSF with a single shared shape, while the Sersic fits use a double Gaussian with independent shape parameters for each component.
 Sersic model fits also include a free centroid, with all other structural parameters shared across all bands.
 That is, the intrinsic model has no color gradients, but the convolved model may have color gradients if the \gls{PSF} parameters vary significantly between bands.
 
-Further details on the performance of these algorithms can be found in \ref{ssec:fluxes}.
+Further details on the performance of these algorithms can are in \secref{ssec:fluxes}.
 
 %%%%%%%%%%%
 \subsection{Variability Measurement}
 
 % Eric
 \subsubsection{Difference Imaging Analysis
 \label{ssec:difim_analysis}}
-Difference Image Analysis (DIA) used the decorrelated Alard \& Lupton image differencing algorithm \citep{DMTN-021}.
+Difference Image Analysis (DIA) uses the decorrelated Alard \& Lupton image differencing algorithm \citep{DMTN-021}.
 We detected both positive and negative \texttt{DIASource} at $5\sigma$ in the difference image.
-Sources with footprints containing both positive and negative peaks were fit with a dipole centroid code.
+Sources with footprints containing both positive and negative peaks due to offsets from the template position or blending were fit with a dipole centroid code.
 
-We filter a subset of \texttt{DIASources} that have pixel flags characteristic of artifacts, non-astrophysical trail lengths, and unphysically negative direct fluxes.
+We filter a subset of \texttt{DIASource}s that have pixel flags characteristic of artifacts, non-astrophysical trail lengths, and unphysically negative direct fluxes.
 We performed a simple spatial association of \texttt{DIASources} into \texttt{DIAObjects} with a one arcsecond matching radius.
 
 % LPG: Do we need this? AP is not part of DP1. Was the reliability model described applied to DP1 data and any data priducts of the model in the DP1 repo?
-To meet the latency requirements for \gls{Alert Production}, we initially developed a relatively simple Machine Learning reliability model: a Convolutional Neural Network with three convolutional layers, and two fully connected layers.
+The Machine Learning reliability model applied to DP1 was developed with the aim to meet the latency requirements for Rubin \gls{Alert Production} when executed on CPUs. Accordingly we developed a relatively simple model: a Convolutional Neural Network with three convolutional layers, and two fully connected layers.
```

Comment:
This gives the impression that this [The Machine Learning model] has already been mentioned, but that's not the case. See suggestion.
```suggestion
A Machine Learning reliability model was developed and applied to DP1 with the aim to meet the latency requirements for Rubin \gls{Alert Production} when executed on CPUs. Accordingly we developed a relatively simple model: a Convolutional Neural Network with three convolutional layers, and two fully connected layers.
```